### DISCUSSION TITLE
BUG - sub links dont work in Header navigation

### DISCUSSION THREAD

**Discussion by shlomof-pyle** [GitbookIO/community#757] [0. General bugs and questions]
hi, some new change to the header navigation section broke something. now anytime i try to add a sublink, it doesnt save or update the preview. i can only get the link to work, not the sublinks. it has destroyed the navigation on my published site.

This is a HIGH priority issues please

this is how it used to look:
![image](https://github.com/user-attachments/assets/9b957f8c-6689-41b1-85f2-00e4cd529abe)
this is how it looks now:
![image](https://github.com/user-attachments/assets/e189fedd-69bb-4b02-8fab-19d29231180f)

**Comment by gregberge** [comment]
We’re working on a fix, and it will be live shortly. We apologize for the inconvenience.

**Reply by shlomof-pyle** [reply]
seems fixed, thank you so much!!

**Comment by shlomof-pyle** [comment]
while we're here, it would be great to allow the header link to just be a dropdown, no link necessary

**Reply by gregberge** [reply]
> while we're here, it would be great to allow the header link to just be a dropdown, no link necessary

Good idea yeah, we will consider it.

**Reply by gregberge** [reply]
Hi @shlomof-pyle,

Just a quick update: the feature is now live and available to everyone! You can now create a dropdown without a link.

Thanks again for your valuable suggestion!
